package.json

```diff
@@ -1,39 +1,35 @@
 {
   "name": "discord-player-deezer",
-  "version": "1.0.1",
+  "version": "2.0.0-beta.2",
   "description": "A custom extractor made for discord-player that enables you to extract from Deezer.",
-  "main": "dist/DeezerExtractor.js",
+  "main": "dist/index.js",
   "author": "retro_ig",
   "license": "MIT",
   "private": false,
   "dependencies": {
-    "@mithron/deezer-music-metadata": "^1.0.3",
-    "soundcloud.ts": "^0.5.2",
-    "tslint-config-prettier": "^1.18.0",
-    "youtube-sr": "^4.3.11"
+    "@mithron/deezer-music-metadata": "^1.0.3"
   },
   "devDependencies": {
-    "@discord-player/extractor": "^4.4.4",
-    "@distube/ytdl": "^2.0.6",
-    "discord-player": "*",
-    "play-dl": "^1.9.7",
-    "prettier": "^2.8.4",
+    "@discord-player/extractor": "^4.5.1",
+    "cross-env": "^7.0.3",
+    "discord-player": "^6.7.1",
+    "prettier": "^3.4.2",
     "tslint": "^6.1.3",
-    "typescript": "^4.9.5",
-    "youtube-ext": "^1.1.25",
-    "yt-stream": "^1.4.14",
-    "ytdl-core": "^4.11.5"
+    "tslint-config-prettier": "^1.18.0",
+    "tsup": "^8.3.5",
+    "typescript": "^5.7.2"
   },
   "scripts": {
-    "build": "yarn lint && tsc",
+    "build": "yarn lint && tsup",
     "lint": "tslint -p tsconfig.json",
-    "npm:publish": "yarn build && npm publish --access public"
+    "npm:publish": "cross-env NODE_OPTIONS=--openssl-legacy-provider yarn build && npm publish --access public"
```

Comment:
why is `cross-env NODE_OPTIONS=--openssl-legacy-provider` needed to publish?

Reply:
I just forgot to remove it. tsup was throwing some errors so I tried using that.

Also, anyone who uses this project needs `--openssl-legacy-provider` when they run their code as decryption for Deezer is somewhat broken.

Anyways, it should be fine to remove it.